src/utf8_fix_test.cc

```diff
@@ -45,7 +45,7 @@ INSTANTIATE_TEST_SUITE_P(Size, FixUtf8StringTest, ::testing::Range(0, 10));
 
 TEST_P(FixUtf8StringTest, FixUtf8String) {
   RandomEngine random(GetParam());
-  std::uniform_int_distribution<uint8_t> random8(0, 0xFF);
+  std::uniform_int_distribution<> random8(0, 0xFF);
```

Comment:
Why this change?

Reply:
if not utf8 then std::string::value_type?

Reply:
According to https://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution:

>  The result type generated by the generator. The effect is undefined if this is not one of short, int, long, long long, unsigned short, unsigned int, unsigned long, or unsigned long long

MSVC also fails to compile this with uint8_t as the template argument.

Should I cast to uint8_t wherever the random8 generator is used here and elsewhere?

Reply:
thanks for explanantion
it's fine with implicit cast